src/config/subgraph.ts

```diff
@@ -0,0 +1,11 @@
+export const SUBGRAPH_URLS = {
+  chainLink: "https://api.thegraph.com/subgraphs/name/deividask/chainlink",
+
+  arbitrumStats: "https://api.thegraph.com/subgraphs/name/gmx-io/gmx-stats",
+  avalancheStats: "https://api.thegraph.com/subgraphs/name/gmx-io/gmx-avalanche-stats",
```

Comment:
i think it make sense to split these values by chainId here (in case it's related to specific chain)? e.g.
```
{
  [ARBITRUM]: {
    stats: "https://api.thegraph.com/subgraphs/name/gmx-io/gmx-stats",
    referrals: ""https://api.thegraph.com/subgraphs/name/gmx-io/gmx-arbitrum-referrals",
    // ...
  },
  [AVALANCHE]: { 
    // ...
  },
  // ...
}
```

if there are a lot of places to update we can do it next time

Reply:
This is not used directly, getting the right by chainId is done in lib/subgraph/clients, also there is additional subgraphs that do not depend on chainId (nissohVault, chainLink)

I think that It's unnecessary right now Because it would complicate the syntax: 
`createClient(SUBGRAPH_URLS[ARBITRUM].stats)`

<img width="792" alt="Screenshot 2022-09-29 at 22 41 22" src="https://user-images.githubusercontent.com/110596516/193115862-d01717f9-1250-4c4d-bb30-fb409bec850a.png">